third_party/terraform/website/docs/r/storage_bucket.html.markdown

```diff
@@ -101,7 +101,7 @@ The following arguments are supported:
 
 * `requester_pays` - (Optional, Default: false) Enables [Requester Pays](https://cloud.google.com/storage/docs/requester-pays) on a storage bucket.
 
-* `bucket_policy_only` - (Optional, Default: false) Enables [Bucket Policy Only](https://cloud.google.com/storage/docs/bucket-policy-only) access to a bucket.
+* `uniform_bucket_level_access` - (Optional, Default: true) Enables [Uniform bucket-level access](https://cloud.google.com/storage/docs/uniform-bucket-level-access) access to a bucket.
```

Comment:
Was the documented default wrong before? The docs about it certainly make it sound like you have to opt in.

Reply:
My bad, i set it back to false.